go-controller/pkg/ovn/loadbalancer/loadbalancer.go

```diff
@@ -83,7 +100,7 @@ func EnsureLBs(nbClient libovsdbclient.Client, externalIDs map[string]string, LB
 		mapLBDifferenceByKey(removeLBsFromGroups, existingGroups, wantGroups, blb)
 	}
 
-	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, lbs...)
+	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, desired...)
```

Comment:
One other performance improvement here is to use a separate `CreateLoadBalancersOps` to directly create the LBs we know don't exist as we did not find them in the search above, avoiding doing the useless  lookup that `CreateOrUpdateLoadBalancersOps` is going to do for them.

Reply:
AIUI, by setting `lb.UUID`, this will short-circuit that.